tox.ini

```diff
@@ -30,5 +30,11 @@ flake8-ignore =
     E501
     W503
     C901
+    py3status/modules/*.py W605 W504 E999
```

Comment:
>We also report one extra error: E999. We report E999 when we fail to compile a file into an Abstract Syntax Tree for the plugins that require it.

From what I understand, this can happen if we mix Python versions with Flake8 versions. I kinda want to try this without E999.

Reply:
You do know I test stuff?

When we do a proper fix of the modules this will be removed.  It is a stop gap (we can continue to work on the project) while we fix things.  We need it for 3.6

Reply:
>You do know I test stuff?

Yes. I believe you. I tried to see if trigger this error with pyenv 3.6.0 and 3.6.6. Couldn't.